content/influxdb/cloud-dedicated/query-data/influxql/parameterized-queries.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Use parameterized queries with InfluxQL
+description: >
+  Use parameterized queries to prevent injection attacks and make queries more reusable.
+weight: 404
+menu:
+  influxdb_cloud_dedicated:
+    name: Parameterized queries
+    parent: Query with InfluxQL
+    identifier: parameterized-queries-influxql
+influxdb/cloud-dedicated/tags: [query, security, influxql]
+list_code_example: |
+  ```go
```

Comment:
It doesn't just have to be code here and I think you should include a title specifying that it's a Go example (alluding that other languages are available).

~~~suggestion
  ##### Go example

  ```go
~~~